test/src/receiver/receiver.cpp

```diff
@@ -76,7 +82,7 @@ using logger_top=cadmium::logger::multilogger<log_messages, global_time>;
 /********************************************/
 /****** APPLICATION GENERATOR *******************/
 /********************************************/
-string input_data_control = "../test/data/receiver_input_test.txt";
+string input_data_control = "test/data/receiver_input_test.txt";
```

Comment:
Define a constant using #define at the top and use the #define value. Do not hardcode.
Do this in all test files where ever applicable.

Reply:
I guess we were not supposed to touch the simulator. The instructor told us to leave the files as such. Also, none of us took this file for any of the tasks.

Reply:
Ok